workspace.jsonc

```diff
@@ -8,7 +8,7 @@
     // "vendorDirectory": "vendor",
   },
   "teambit.dependencies/dependency-resolver": {
-    "packageManager": "teambit.dependencies/pnpm",
+    "packageManager": "teambit.dependencies/yarn",
```

Comment:
@ranm8 , revert this change, otherwise, many tests will fail.

Reply:
Thanks!